porogo/frontend/static/styles/views/_home.scss

```diff
@@ -1,44 +1,38 @@
 
 @import '../common/variables';
 
-.top-content {
-    height: $body-height;
-}
 
 .top-content, .full-content {
     display: flex;
     align-items: center;
     justify-content: center;
-}
-
-.full-content {
     height: 100%;
 }
 
+
 #landing {
     height: 100%;
-    width: 100%;
-
     display: flex;
     align-items: center;
-    justify-content: space-evenly;
-
-    background: linear-gradient(rgba(0,0,0,0) 0%, rgba(0,0,0,0) 80%, rgba(162, 193, 224, 0.5) 100%);
```

Comment:
Are we dropping the little gradient at the bottom?

Reply:
Yeah I don't remember why I added that to begin with. Can check to make it more stylish another time.